src/pages/SignIn.tsx

```diff
@@ -33,6 +35,18 @@ export default function SignIn(): ReactElement {
             key: VITE_FORT_MATIC_KEY, // required
           },
         },
+        portis: {
+          package: Portis, // required
+          options: {
+            id: "a180917a-e8d2-43a9-a782-456579bf39b9", // required
```

Comment:
@unicorn-talent  - what is this ID used for relative to Portis?

Reply:
we need to register ID in Portis